spec/models/metrics/rejection_percentage_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Metrics::RejectionPercentage do
+  Rejection::REASONS.each do |reason|
+    it "##{reason} defaults to 0" do
+      expect(subject.public_send(reason)).to eq(0)
+    end
+  end
+
+  describe '#attributes' do
```

Comment:
@alan that model doesn't calculate anything. It really just is a DAO. 
I could test that setting a value sets a value.
All the computation is tested in ```GraphMetricsPresenter``` specs
Or maybe i'm miss understanding what you're saying